Update events in render methods of the bVirtualScroll (#678)
V4Fire/Client: src/base/b-virtual-scroll/CHANGELOG.md

> - :house:      [Internal]
> - :nail_care:  [Polish]

## v3.15.4 (2021-01-24)

#### :rocket: New Feature

* Added a new events `chunkRender:*`

> #### :boom: Breaking Change

Reviewer: Должно быть выше

> * `chunkRenderStart` is renamed and now will be emitted before the component driver renders component

Reviewer: ```suggestion
* The event`chunkRenderStart` is renamed to `chunkRender:renderStart` and now it emits before a component driver renders components
```
